.travis.yml

```diff
@@ -5,7 +5,7 @@ jobs:
   include:
     
     - name: Build and push docker image with the newest kube-version
-      if: type = cron
+      if: type = cron AND branch = master
```

Comment:
hmm, isn't this an `OR` condition? I would guess that we need to specify the branch in travis-ci on which the cron will run on

Reply:
The `branch = master` is not strictly necessary if we specify the branch, but with `OR` the job would also run on push events which is not intended.